### PR TITLE
Improve command line run documentation

### PR DESCRIPTION
**Pull request by jvegasbsc**
Before you start, please read our [contribution guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html).

**Tasks**

-   [x] This pull request has a descriptive title that can be used in a changelog

### CODE REVIEW

**Comment by jvegasbsc** [comment]
> Shouldn't we document this also here?

In that case I should add dlocumentation for all the parameters. I will do it tomorrow

**Comment by bouweandela** [comment]
@jvegasbsc If you have time, could you please have a look at this?

**Comment by bouweandela** [comment]
Related to #805. @jvegasbsc Is this ready for review now?

**Comment by jvegasbsc** [comment]
> @jvegasbsc Is this ready for review now?

It is just documentation, so you can left this for the last minute, just in case I find a way to generate it automatically before monday

**Comment by jvegasbsc** [comment]
No way found, so please check and merge

File: doc/quickstart/run.rst [hunk 1]
```diff
@@ -25,13 +25,63 @@ It is also possible to explicitly change values from the config file using flags
...
+	esmvaltool run --max_datasets=NDATASETS --max_years=NYEARS recipe_python.yml
+
+In this case, the recipe will limit the number of datasets per variable to
+NDATASETS and the total amount of years loaded to NYEARS. They can also be used
+separately.
+
+
+.. code:: bash
+
+    esmvaltool run --skip_nonexistent=True recipe_python.yml SKIP_NONEXISTENT
```
Comment (bouweandela) [inline doc/quickstart/run.rst:55]:
```suggestion
To run a recipe, even if some datasets are not available, use

.. code:: bash

    esmvaltool run --skip_nonexistent=True recipe_python.yml
```

File: doc/quickstart/run.rst [hunk 2]
```diff
@@ -25,13 +25,63 @@ It is also possible to explicitly change values from the config file using flags
...
+
+
+If Synda is installed (see http://prodiguer.github.io/synda/), it is possible
+to use it to automatically download the requested data from ESGF if it is not
+available locally:
+
+.. code:: bash
+
+    esmvaltool run --synda_download True recipe_python.yml
```
Comment (bouweandela) [inline doc/quickstart/run.rst:64]:
```suggestion
    esmvaltool run --synda_download=True recipe_python.yml
```

**Review by bouweandela — APPROVED** [review]

**Pull request merged**